### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by ndavison**
A large portion of this PR is actually just code styling changes as a result of a version increase in eslint. The files that actually have code changes as a result of updated APIs etc are:

* `apps/marketplace/components/Opportunities/`
* `apps/marketplace/components/BuyerDashboard/`
* `apps/marketplace/components/Header/Header.js`

### CODE REVIEW

File: apps/marketplace/components/Opportunities/OpportunitiesFilters.js [hunk 1]
```diff
@@ -394,12 +395,12 @@ export class OpportunitiesFiltersComponent extends BaseForm {
...
                 </div>
                 <span className={styles.cancelLink}>
-                  <a href="#cancel" onClick={this.handleFilterCancelClick}>
+                  <a href="#cancel" data-type="" onClick={this.handleFilterCancelClick}>
                     Cancel
                   </a>
                 </span>
                 <span className={styles.applyFilters}>
-                  <a href="#apply" onClick={this.handleFilterApplyClick}>
+                  <a href="#apply" data-type="" onClick={this.handleFilterApplyClick}>
```
Comment (tobiasbrown) [inline apps/marketplace/components/Opportunities/OpportunitiesFilters.js]: Do we need these empty attributes?

File: apps/marketplace/components/Opportunity/OpportunityInfoCard.js [hunk 2]
```diff
@@ -24,118 +23,110 @@ const OpportunityInfoCard = props => (
...
+                    <span>
+                      {' '}
+                      You must also add {props.category} as a category you provide services for to apply for this brief.
+                    </span>
+                  )}
+                </span>
               )}
-            </span>
-          )}
+              {}
```
Comment (tobiasbrown) [inline apps/marketplace/components/Opportunity/OpportunityInfoCard.js]:
```suggestion
```

File: package.json [hunk 3]
```diff
@@ -138,7 +137,7 @@
     "server": "node scripts/server.js",
     "server:development": "nodemon $NODE_DEBUG_OPTION scripts/server.js --watch build/render-server.js",
     "bdd-test": "node ./node_modules/.bin/wdio wdio.conf.js",
-    "eslint": "./node_modules/.bin/eslint .",
+    "eslint": "./node_modules/.bin/eslint apps",
```
Comment (malmas18) [inline package.json]: wouldn't it be better to use the eslintignore?

folders like fixtures is ignored now

**Comment by ndavison** [comment]
> this looks fine.
> 
> was this tested with `npm run prettier`?

Yep - it's what I used to enforce the new linting rules ;)

File: apps/marketplace/components/Opportunity/Opportunity.js [hunk 4]
```diff
@@ -1,12 +1,10 @@
 import React from 'react'
-import { Redirect } from 'react-router-dom'
-import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
 import AUheading from '@gov.au/headings/lib/js/react.js'
 import format from 'date-fns/format'
 import { rootPath } from 'marketplace/routes'
 import NotVisible from 'marketplace/components/Icons/NotVisible/NotVisible'
-import { getBriefLastQuestionDate, hasPermission } from 'marketplace/components/helpers'
+import { getBriefLastQuestionDate } from 'marketplace/components/helpers'
```
Comment (malmas18) [inline apps/marketplace/components/Opportunity/Opportunity.js]: permission checks?

File: package.json [hunk 5]
```diff
@@ -142,6 +141,7 @@
     "prettier": "./node_modules/.bin/prettier --print-width 120 --single-quote --no-semi --write \"apps/**/*.js\"  --write \"fixtures/**/*.js\""
   },
   "jest": {
+    "verbose": false,
```
[the remark this answers is not in the document]
Comment (ndavison) [reply package.json]: Strangely this seems to be required to get jest tests to print out console statements in test failures now.

**Pull request merged**